### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#322] [Ideas]
Is anyone interested into having a separate, more chill Discord server for discussion outside of this repo ? I know the idea was given before by another user, but since we're in here, in a way more secure space, tell me if you're interested with this idea.

**Comment by BellezaEmporium** [comment]
-snip- I've created one server, that will be made for discussions outside of GH (as an example, it'll be easier to exchange on a discussion to give a team answer instead of an individual answer).

It'll also contain some more chilled chats, in case you wanna talk about rain and sunshine. Feel free to join if you feel like it.

**Comment by BellezaEmporium** [comment]
> -snip- I've created one server, that will be made for discussions outside of GH (as an example, it'll be easier to exchange on a discussion to give a team answer instead of an individual answer).
> 
> It'll also contain some more chilled chats, in case you wanna talk about rain and sunshine. Feel free to join if you feel like it.

It also contains some programs I've made too, like the BTV scraper.

**Comment by freearhey** [comment]
The discussion has been moved to _iptv-org/iptv-org.github.io_ repository due to GitHub's decision to disable "Team Discussions": https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/
